marathon_acme/tests/test_sse_protocol.py

```diff
@@ -0,0 +1,285 @@
+# -*- coding: utf-8 -*-
+from testtools import TestCase
+from testtools.matchers import Contains, Equals, IsInstance
+
+from twisted.internet.defer import Deferred
+from twisted.internet.error import ConnectionLost
+
+from marathon_acme.sse_protocol import SseProtocol
+
+
+class TestSseProtocol(TestCase):
+
+    def setUp(self):
+        super(TestSseProtocol, self).setUp()
+        self.protocol = SseProtocol()
+
+        class DummyTransport(object):
+            disconnecting = False
```

Comment:
Is there any particular reason this class needs to be defined in the `setUp()` method? If not, it's better to do it at module level.